hack/ccm-canary-entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
This script is for the new entrypoint. The easiest solution (rather than mess with GOPATH) is for the entrypoint script to move to project root and execute the canary test from there.